content/blog/setting-bios-and-storage-controller-properties-with-redfish.md

```diff
@@ -15,88 +16,40 @@ tags:
-
-![PUT Return Status](/img/2-put-return-status.png "PUT Return Status")
-
-At this stage of the process, only the staging / "pending zone" of the Smart Array has changed and contains the `PUT` payload. We can verify this assertion by comparing the content of  `{{Subsystem-URI}}/Settings/` with the content of `{{Subsystem-URI/}}`.
-
-In the pending zone (`.../SmartStorageConfig/Settings/`) we can see the payload we sent to the Redfish server. However, in the "running zone" (`.../SmartStorageConfig/`) the `LogicalDrives` array is still empty:
-
-![GET Pending and Current LogicalDrives](/img/3-getpendinglogicadrives.bmp "GET Pending and Current LogicalDrives")
-
-It is now time to reset the server and perform a `GET` of the running zone. In the response body of this operation, the first Redfish object is a `@Redfish.Settings` collection containing a single `MessageID` mentioning `Success`. This single message is synonym of a successful transfer of the "pending zone" in to the "running zone". We will see later in this document what we get in case of an un-successful transfer.
-
-![GET Running zone after server reset](/img/5-getafterreset.png "GET Running zone after server reset")
-
-Further down in this response we find the `LogicalDrives` array containing the Raid1 disk array:
-
-![GET LogicalDrives from running zone](/img/6-getafterreset-2.png "GET LogicalDrives from running zone")
-
-## Unsuccessful example
-
-In order to emphasize the fact that the modification status of properties in the `Bios` and `SmartStorage` subsystems must be done after a system reset, we will study a case where the JSON `PUT` payload is syntactically correct but embedding a value typo (`Raid` instead of `Raid1`) and missing a required key-value (`DataGuard=Disabled`):
-
-![PUT of a bad payload](/img/7-badpayload.png "PUT of a bad payload")
-
-Sending this request returns an HTTP `200 OK` status and a `SystemResetRequired` Redfish message just like in the previous example:
-
-![PUT return status of bad Redfish request](/img/8-put-return-status-of-bad-request.png "PUT return status of bad Redfish request")
-
-The pending zone contains the faulty payload:
-
-![Faulty payload in pending zone](/img/9-pendingzonewithbadrequest.png "Faulty payload in pending zone")
-
-After the server reset, a `GET` of the running zone responds with a `MessageArgs=[DataGuard]` object and two `MessageID`keys. The first one mentions `DataGuard` as a missing property and the second one (`Success`) means that the analysis of the transfer from the pending zone to the running zone has successfully ended. This `Success` message does not means that the transfer has occurred.
-
-Moreover, note that there is nothing mentioning the `Raid` typo. It means that analysis of the payload to transfer stops at the first error found.
```

Comment:
change "mentioning" to "that mentions", so it reads "Moreover, note that there is nothing that mentions the 'Raid' typo."